package.json

```diff
@@ -22,15 +22,12 @@
     "watch":
       "babel --plugins transform-es2015-modules-umd src --watch --ignore tests --out-dir ./dist",
     "build": "babel --plugins transform-es2015-modules-umd src --ignore tests --out-dir ./dist",
-    "lint": "eslint 'src/**/*.js'",
```

Comment:
The same case here, you can't remove it. This ensures the style and quality code